docs/src/pages/index.tsx

```diff
@@ -234,38 +391,60 @@ function IntegrationMembranePlugin() {
           <div className="card__body container">
             <p>
               Membrane is a developer-friendly multimedia framework for Elixir. You can easily add
-              video composing functionality into your multimedia pipeline using Membrane Live
-              Compositor Plugin.
+              video and audio composing functionality into your multimedia pipeline using Membrane
+              LiveCompositor Plugin.
             </p>
-            <p>For more, see:</p>
-            <p>
-              <FaFile style={{ fontSize: 15, marginRight: 7 }} />
-              Documentation -{' '}
-              <Link href="/docs/intro#membrane-framework-plugin">
-                Get started with Membrane plugin
+            <div className="row margin--sm" style={{ justifyContent: 'center' }}>
+              <div className="col" />
+              <Link
+                className={clsx(
+                  'button',
+                  'button--primary',
+                  'button--lg',
+                  'margin--sm',
+                  'col',
+                  styles.hoverPrimary
+                )}
+                to="https://hexdocs.pm/membrane_video_compositor_plugin/Membrane.VideoCompositor.html">
+                <IconContext.Provider value={{ style: { verticalAlign: 'middle' } }}>
+                  <FaBook style={{ marginRight: 5 }} />
+                  Docs
+                </IconContext.Provider>
               </Link>
-            </p>
-            <p>
-              <FaGithub style={{ fontSize: 15, marginRight: 7 }} />
-              GitHub repository -{' '}
-              <Link href="https://github.com/membraneframework/membrane_live_compositor_plugin">
-                membraneframework/membrane_live_compositor_plugin
+              <Link
+                className={clsx(
+                  'button',
+                  'button--secondary',
+                  'button--outline',
+                  'button--lg',
+                  'margin--sm',
+                  'col',
+                  styles.hoverSecondary
+                )}
+                to="https://github.com/membraneframework/membrane_live_compositor_plugin">
+                <IconContext.Provider value={{ style: { verticalAlign: 'middle' } }}>
+                  <FaGithub style={{ marginRight: 5 }} />
+                  Plugin repo
```

Comment:
I would avoid abbreviations like repo, at least on the main page. 

Plugin repository/Sources/Repository/View on GitHub